agent/api/task/taskvolume_test.go

```diff
@@ -163,15 +165,17 @@ func TestInitializeSharedProvisionedVolume(t *testing.T) {
 
 	// Expect the volume already exists on the instance
 	dockerClient.EXPECT().InspectVolume(gomock.Any(), gomock.Any(), gomock.Any()).Return(dockerapi.VolumeResponse{})
-	err := testTask.initializeDockerVolumes(dockerClient, nil)
+	err := testTask.initializeDockerVolumes(SharedVolumeMatchAll, dockerClient, nil)
```

Comment:
nit : can just pass in `true` instead of new variable

Reply:
I prefer this way since it provides more semantics to the test; true/false value on its own does not show what it is for.